doc/examples/chrome-debugging-protocol/README.md

```diff
@@ -0,0 +1,12 @@
+# axe-chrome-debugging-protocol-example
+
+This (very minimal) example demonstrates how to use `axe-core` with the [Chrome Debugging Protocol](https://chromedevtools.github.io/devtools-protocol/).
+
+The example does not have feature pairty with [`axe-webdriverjs`](https://github.com/dequelabs/axe-webdriverjs), and does not run on `<iframe>`s.
+
+## To run the example
+
+- Ensure Node v8+ is installed and on `PATH`
+- Move to the `doc/examples/chrome-debugging-protocol` directory
+- Run `npm install`
+- Run `node axe-cdp.js http://www.deque.com` to run `axe-core` via Puppeteer against http://www.deque.com and output results to the terminal
```

Comment:
I'm getting a conflicting port error even after a machine reboot:
```
Error running axe-core: connect ECONNREFUSED 127.0.0.1:9222
```

Reply:
Good call, we need to start `google-chrome` with that port opened. Will update.